stable/stolon/templates/keeper-statefulset.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: apps/v1beta2
+kind: StatefulSet
+metadata:
+  name: {{ template "stolon.fullname" . }}-keeper
+  labels:
+    app: stolon-keeper
+    chart: {{ template "stolon.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: keeper
+spec:
+  serviceName: {{ template "stolon.fullname" . }}-keeper
+  replicas: {{ .Values.keeper.replicaCount }}
+  selector:
+    matchLabels:
+      app: stolon-keeper
+      release: {{ .Release.Name }}
+      component: keeper
+      stolon-cluster: {{ template "stolon.fullname" . }}
```

Comment:
What is label `stolon-cluster` needed for? Anyways, it is not required to be part of selectors.